fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
+  $('.jsSelectedMainEmail').text(postVariable.email);
```

Comment:
Codacy found an issue: ['postVariable' is not defined. (no-undef)](https://app.codacy.com/manual/prodcodacytestuser/CheckPrCommentsPublicProjectProd/pullRequest?prid=5766805)